src/firmware/app/world/chicker.h

```diff
@@ -9,18 +9,16 @@ typedef struct Chicker Chicker_t;
 /**
  * Create a chicker with the given functions for interacting with it
  *
- * @param kick A function that we can call to kick with the chicker with a given speed,
- *             in meters per second
- * @param chip A function that we can call to chip a given distance, in meters, with the
- *             chicker. The distance parameter is the distance to the first bounce.
- * @param enable_autokick A function that we can call to enable autokick on the chicker
- *                        (ie. the chicker will kick the ball as soon as the ball is close
- *                        enough to the front of the chicker to be kicked), with a kick
- *                        speed in meters per second
- * @param enable_autochip A function that we can call to enable autochip on the chicker
- *                        (ie. the chicker will chip the ball as soon as the ball is close
- *                        enough to the front of the chicker to be chipped), with a chip
- *                        distance in meters
+ * @param kick A function that we can call to kick with the chicker with a given
+ * speed, in meters per second
+ * @param chip A function that we can call to chip a given distance, in meters, with
+ * the chicker. The distance parameter is the distance to the first bounce.
+ * @param enable_autokick A function that we can call to enable autokick on the
+ * chicker (ie. the chicker will kick the ball as soon as the ball is close enough to the
+ * front of the chicker to be kicked), with a kick speed in meters per second
+ * @param enable_autochip A function that we can call to enable autochip on the
+ * chicker (ie. the chicker will chip the ball as soon as the ball is close enough to the
+ * front of the chicker to be chipped), with a chip distance in meters
```

Comment:
wondering why this was changed

Reply:
Initially added [in/out] specifiers but because function pointers don't need specifiers, they were removed.

Reply:
can we just revert these changes then?

Reply:
Done.